src/main/java/org/swisspush/redisques/QueueStatsService.java

```diff
@@ -186,7 +190,7 @@ private <CTX> void fetchRetryDetails(GetQueueStatsRequest<CTX> req, BiConsumer<T
         });
     }
 
-    private <CTX> void attachDequeueStats(GetQueueStatsRequest<CTX> req, BiConsumer<Throwable, GetQueueStatsRequest<CTX>> onDone) {
+    <CTX> void attachDequeueStats(GetQueueStatsRequest<CTX> req, BiConsumer<Throwable, GetQueueStatsRequest<CTX>> onDone) {
```

Comment:
Do not change visibility of methods/variables just to test them. This is bad practice. If you can find a workaround or there is a arguable reason, it may be ok.

Reply:
ok, I will rollback this and implement a tests in another way.